### ISSUE TITLE
Internal error when changing user username

### ISSUE THREAD

**Issue by mjovanovic0** [RocketChat/Rocket.Chat#11269]
### Description:

When I try to change username for user I get internal server error.

### Steps to reproduce:

1. Go to Admin panel
2. Go to Users
3. Find user you want to edit
4. Edit user
5. In username field change username to new one
6. Save
7. Exception occure

### Expected behavior:

Username for user successfully changed.

### Actual behavior:

Following exception:
Exception while invoking method 'insertOrUpdateUser'
```javascript
MongoError: The positional operator did not find the match needed from the query.
    at Function.MongoError.create (/app/bundle/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/error.js:31:11)
    at toError (/app/bundle/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb/lib/utils.js:139:22)
    at /app/bundle/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb/lib/collection.js:1059:67
    at /app/bundle/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/connection/pool.js:469:18
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
```

### Server Setup Information:

- Version of Rocket.Chat Server: 0.65.2
- Operating System: Ubuntu 16.04.4 LTS up-to-date
- Deployment Method: docker
- Number of Running Instances: 1
- DB Replicaset Oplog: Disabled
- NodeJS Version: v8.9.3 - x64
- MongoDB Version: v3.6.5

**Comment by TwizzyDizzy** [comment]
I cannot confirm this on Rocket.Chat `0.66.0`. I'm using MongoDB 2.6 though, so it might be related to Mongo 3.x?

Cheers
Thomas

**Comment by TwizzyDizzy** [comment]
Just found #11155, so I'm closing this issue. Please add all future information over there! Thanks :)

Cheers (+close)
Thomas

**Comment by mjovanovic0** [comment]
This issue is fixed in 0.66.1 with MR #11335 for #11155
